packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -304,6 +308,8 @@ const RelationInput = ({
             {({ data, index, style }) => {
               const { publicationState, href, mainField, id } = data[index];
               const statusColor = publicationState === 'draft' ? 'secondary' : 'success';
+              // Use Box to fix the issue of BoxEllipsis causing LinkEllipsis :focus-visible:after "outline" hidden
+              const BoxWrapper = href ? Box : BoxEllipsis;
```

Comment:
Hi @gu-stav,  
This resolve the highlighted problem mentioned in the linked issue, for use cases where the item title is a `Link` element.

For the other use case with `Typography`, where we have a text only / non-clickable element, would love to get your thoughts on if it's desired to have outline made visible there as well?, since it isn't ideal from accessibility standpoint to tab/outline text only elements.

Reply:
hi @gitstart!
the outline/tabbing option of Typography is provided there because of the tooltip which is there to make sure we can access the full title of the relation item 👀

Reply:
Thanks @ronronscelestes So the Tooltip is focusable, let's us try to fix the its outline in this case too.